src/Compilers/CSharp/Test/Semantic/CSharpCompilerSemanticTest.csproj

```diff
@@ -12,6 +12,8 @@
     <RoslynProjectType>UnitTest</RoslynProjectType>
   </PropertyGroup>
   <ItemGroup Label="Project References">
+    <ProjectReference Include="..\..\..\..\Scripting\Core\Scripting.csproj" />
```

Comment:
Let's avoid adding dependency on Scripting to the compiler tests.

You can set up the tests similarly to what this test is doing:
http://source.roslyn.io/#Roslyn.Compilers.CSharp.Emit.UnitTests/CodeGen/CodeGenScriptTests.cs,390

`CreateSubmission` test helper allows you to create a script submission compilation with a global type.

Reply:
Agree. Definitely need to remove this.

Reply:
I refactored to use CreateSubmission and VerifyEmitDiagnostics as in the example you linked, but this make the tests stop failing. What did I miss? https://github.com/jnm2/roslyn/commit/39fd8031c29f43558be08d4e89a4aaae72246cf2#diff-cc8ba01454b19ec8f0c694fe2b25842d

Reply:
@tmat I refactored to use CreateSubmission and VerifyEmitDiagnostics as in the example you linked, but this made the tests stop failing. What did I miss? https://github.com/jnm2/roslyn/commit/39fd8031c29f43558be08d4e89a4aaae72246cf2#diff-cc8ba01454b19ec8f0c694fe2b25842d

Reply:
Oh! I'm just impatient, that's all. The errors didn't show up until ` 5 running,  0 queued, 47 completed` and I thought I remembered them showing up in the first ten assemblies tested. All good!